generation/um/windows.data.pdf.interop/generate.rsp

```diff
@@ -0,0 +1,24 @@
+@../../settings.rsp
+@../../remap.rsp
+--file
+um-windows.data.pdf.interop.h
+--output
+../../../sources/Interop/Windows/um/windows.data.pdf.interop
+--test-output
+../../../tests/Interop/Windows/um/windows.data.pdf.interop
+--traverse
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.19041.0/um/windows.data.pdf.interop.h
+--with-librarypath
+*=Windows.Data.Pdf
+--with-attribute
+IPdfRendererNative=Guid("7d9dcd91-d277-4947-8527-07a0daeda94a")
+--exclude
```

Comment:
nit: `--exclude` should appear before `--file` and be alphabetically sorted, ignoring case, same as the other generate.rsp files.